server/migrations/versions/cea4d948af0d_initial_migration.py

```
@@ -0,0 +1,202 @@
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('client_id', sa.Integer(), nullable=False),
+    sa.Column('admin_id', sa.Integer(), nullable=False),
+    sa.Column('subject', sa.String(length=255), nullable=False),
+    sa.Column('status', sa.Enum('OPEN', 'CLOSED', 'IN_PROGRESS', name='ticketstatus'), nullable=False),
+    sa.Column('created_at', sa.DateTime(timezone=True), nullable=False),
+    sa.ForeignKeyConstraint(['admin_id'], ['users.id'], ),
+    sa.ForeignKeyConstraint(['client_id'], ['users.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('tokens',
+    sa.Column('id', sa.Integer(), nullable=False),
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)